plugins/janus_sip.c

```diff
@@ -4916,3 +4967,9 @@ gpointer janus_sip_sofia_thread(gpointer user_data) {
 	return NULL;
 }
 
+static void update_session_status(janus_sip_session *session, janus_sip_call_status new_status) {
```

Comment:
Why not avoid the declaration above, and just define the method directly there right away? It's small enough, and the declaration isn't really needed.

As a side note, I guess you added this method to track the chanegs when debugging?

Reply:
Agree. And, correct, to make it easier to follow the session state, which is useful beyond just debugging. I hope we can keep the output?